.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Description
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Paste an example playbook that can be used to reproduce the problem and/or describe steps to reproduce the behavior.
+
+## Current Behavior
```

Comment:
Would you like to capitalize the headings according to titles or keep the "capitalize only the first letter" format?

```suggestion
## Current behavior
```